tests/crypto/sse/findex/implementations/redis/tests/findex.test.ts

```diff
@@ -33,7 +33,62 @@ test("upsert+search", async () => {
       LABEL,
       "france",
       false,
-      1000
+      1000,
+      1000,
+      (res: Uint8Array[]) => {
+        console.log(res);
```

Comment:
Maybe add a test to check if the `progress` callback receive all the element by incrementing a counter and comparing with the final result?